src/lib/telldus-accessory.js

```diff
@@ -282,6 +282,8 @@ class TelldusThermometer extends TelldusAccessory {
 
     controllerService.getCharacteristic(
       this.Characteristic.CurrentTemperature
+    ).setProps(
+      { minValue: -50 }
```

Comment:
Cool, I did not know about this:)

Please consider adding a comment about why this is here, like -50 degrees being the coldest one could expect from a thermometer.